### PR TITLE
Switch testing to Docker

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by rhornung67** [comment]
@davidbeckingsale this is the same behavior I saw last week when I tried to switch over to use the RAJA Docker build....Let's try to work this out tomorrow.

**Comment by davidbeckingsale** [comment]
Yeah, very odd. I have told @trws about it too.

**Comment by rhornung67** [comment]
@davidbeckingsale did you set up the Docker configuration for RAJA?  I noticed there is an access key provided in the travis configuration for RAJA that as your name on it. It isn't in the RAJAPerf configuration.  Here's the link https://github.com/LLNL/RAJA/settings/hooks/8330109.

**Comment by davidbeckingsale** [comment]
As far as I know, that's only related to the Intel compiler - shouldn't impact any others...

**Comment by trws** [comment]
Yup, that’s specifically the decryption key to make the intel container usable, it shouldn’t mess with the others.

**Comment by trws** [comment]
Fixed.  The script wasn't a valid executable.

You may want to rebase out the intermediate travis.yml commits, they aren't necessary anymore.

**Comment by davidbeckingsale** [comment]
Wow! Thanks for finding that. We can squash merge this.

File: scripts/travis_build_and_test.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 #!/bin/bash
```
Comment (rhornung67) [inline scripts/travis_build_and_test.sh]: @trws thanks! This was an obvious bonk on my part.  I'm going to crawl into a hole now....

**Comment by rhornung67** [comment]
@davidbeckingsale feel free to merge when tests pass. I'm away from my laptop most of the time while away. I will approve.

**Comment by davidbeckingsale** [comment]
Awesome! Thanks for your help @trws 😄

**Comment by trws** [comment]
No problem, and FWIW @rhornung67, I wouldn't call this one an obvious bonk at all.  I was staring at this one cross-eyed for a good while trying to figure out what was going on. 😄

**Comment by rhornung67** [comment]
@trws nevertheless, your work to straighten this out is much appreciated!

**Pull request merged**